runtime/doc/matchit.txt

```diff
@@ -150,35 +142,12 @@ setting |b:match_skip|.
 ==============================================================================
 2. Activation						*matchit-activate*
 
-You can use this script as a plugin, by copying it to your plugin directory.
-See |add-global-plugin| for instructions.  You can also add a line to your
-|vimrc| file, such as >
-	:source $VIMRUNTIME/macros/matchit.vim
-or >
-	:runtime macros/matchit.vim
-Either way, the script should start working the next time you start up Vim.
-
-(Earlier versions of the script did nothing unless a |buffer-variable| named
-|b:match_words| was defined.  Even earlier versions contained autocommands
-that set this variable for various file types.  Now, |b:match_words| is
-defined in many of the default |filetype-plugin|s instead.)
-
 For a new language, you can add autocommands to the script or to your vimrc
 file, but the recommended method is to add a line such as >
 	let b:match_words = '\<foo\>:\<bar\>'
```

Comment:
Not really, I'll change it to this:
~~~
For a new language, you can add a line such as >
	let b:match_words = '\<foo\>:\<bar\>'
to the corresponding |filetype-plugin|.  See |b:match_words| below for how
this variable is interpreted.
~~~